core/templates/dev/head/components/side_navigation_bar/SideNavigationBarDirective.ts

```diff
@@ -21,13 +21,16 @@ oppia.directive('sideNavigationBar', [
     return {
       restrict: 'E',
       scope: {},
+      bindToController: {},
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/side_navigation_bar/' +
         'side_navigation_bar_directive.html'),
-      controller: ['$scope', '$timeout', function(
-          $scope, $timeout) {
-        $scope.currentUrl = window.location.pathname;
-        $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+      controllerAs: '$ctrl',
+      controller: ['$timeout', function(
+          $timeout) {
+        var ctrl = this;
+        ctrl.currentUrl = window.location.pathname;
```

Comment:
Maybe we should do this separately but just a note that this should probably either make use of $window rather than window, or use UrlService perhaps.

Reply:
Filed an issue #6738